test/src/ariaqueryhandler.spec.ts

```diff
@@ -553,9 +565,14 @@ describe('AriaQueryHandler', () => {
       const {page} = getTestState();
 
       let divFound = false;
-      const waitForSelector = page.waitForSelector('aria/zombo').then(() => {
-        return (divFound = true);
-      });
+      const waitForSelector = page
+        .waitForSelector('aria/zombo')
+        .then(() => {
+          return (divFound = true);
+        })
+        .catch(err => {
+          return err;
```

Comment:
On theory error can have a value `true` so return `false` is the correct thing to do here.

Reply:
Or add the same expect as the previous test `expect(divHidden).toBe(true);`